### PR TITLE
feat: export owlmoji

### PR DESCRIPTION
**Pull request by kellyjosephprice**
| [![PR App][icn]][demo] | Fix RM-XYZ |
| :--------------------: | :--------: |

## 🧰 Changes

Exports our `Owlmoji` class for consumption in the main app.

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

**Review by erunion — APPROVED** [review]

File: processor/parse/gemoji-parser.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const Emoji = require('../../lib/gemoji');
```
Comment (jennspencer) [inline processor/parse/gemoji-parser.js:1]: where did the name `gemoji` come from?? 🤔

**Review by jennspencer — APPROVED** [review]

File: processor/parse/gemoji-parser.js (hunk 1 above)
Comment (erunion) [reply processor/parse/gemoji-parser.js:1]: https://github.com/github/gemoji

Comment (kellyjosephprice) [reply processor/parse/gemoji-parser.js:1]: I was so dissappointed when I learned that it's just **G** (ithub) + **emoji**

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.81.0</b>](/readmeio/markdown/releases/tag/v6.81.0)
<br>

**Pull request merged**